token-lending/js/tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "emitDeclarationOnly": true,
         "outDir": "lib",
         "typeRoots": ["node_modules/@types", "types"],
-        "module": "esnext",
+        "module": "commonjs",
```

Comment:
Sounds good. I changed this after doing some research online on the following error:
```
(Use `node --trace-warnings ...` to show where the warning was created)
/Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/cli/main.ts:1
import { Blockchain } from './blockchain';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:355:18)
```

Any suggestions for how to fix this otherwise? I tried another common suggestion from online, setting `type: module` in `package.json` but that leads to a different error:
```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/ilmoi/Downloads/FORKED_SPL2/token-lending/js/cli/main.ts
```

Sorry I'm new to typescript. I'll keep digging in the meantime.

Reply:
Found a solution - simply include a second, shorter `tsconfig.json` specifically within the `cli` folder:
```
{
    "compilerOptions": {
        "target": "esnext",
        "module": "commonjs",
    },
}

```

Reply:
Ah, nice. Yeah, not sure if it helps here, but you can also have a typescript config extend a root one too: https://www.typescriptlang.org/tsconfig/extends.html